habitat-hitl/habitat_hitl/environment/controllers/gui_controller.py

```diff
@@ -24,27 +23,106 @@
 class GuiRobotController(GuiController):
     """Controller for robot agent."""
 
+    def __init__(
+        self,
+        agent_idx,
+        is_multi_agent,
+        gui_input,
+        articulated_agent,
+        num_actions,
+        base_vel_action_idx,
+        num_base_vel_actions,
+        turn_scale,
+    ):
+        super().__init__(agent_idx, is_multi_agent, gui_input)
+        self._articulated_agent = articulated_agent
+        self._hint_walk_dir = None
+        self._hint_distance_multiplier = None
+        self._cam_yaw = None
+        self._hint_target_dir = None
+        self._base_vel_action_idx = base_vel_action_idx
+        self._num_base_vel_actions = num_base_vel_actions
+        self._turn_scale = turn_scale
+
+        self._actions = np.zeros((num_actions,))
+
+    def set_act_hints(
+        self,
+        walk_dir,
+        distance_multiplier,
+        grasp_obj_idx,
+        do_drop,
+        cam_yaw=None,
+        throw_vel=None,
+        reach_pos=None,
+        hand_idx=None,
+        target_dir=None,
+    ):
+        assert (
+            throw_vel is None or do_drop is None
+        ), "You can not set throw_velocity and drop_position at the same time"
+        self._hint_walk_dir = walk_dir
```

Comment:
This is sloppy. I'm not actually using `_hint_walk_dir` anywhere (yet). Likewise for `_hint_distance_multiplier` and `_hint_target_dir`.